packages/web-wallet/src/pages/TransactionHistory.tsx

```diff
@@ -1,5 +1,91 @@
-function TransactionHistory(): JSX.Element {
-  return <div>TODO: Transaction History</div>;
+import Table from '@components/Table/Table';
+import React from 'react';
+import PageHeading from '@components/PageHeading/PageHeading';
+
+const data = [
+  {
+    recipient: 'Shielded Recipient',
+    amount: '+0.008',
+    status: 'Received',
+    date: 'Oct 31, 2023',
+  },
+  {
+    recipient: 'zs13h7mt2xyk5t2xy38...2xvxser2',
+    amount: '-0.001',
+    status: 'Sent',
+    date: 'Jun 12, 2024',
+  },
+  {
+    recipient: 'zs13h7mt2xyk5t2xy38...2xvxser2',
+    amount: '-0.001',
+    status: 'Sent',
+    date: 'Jun 12, 2024',
+  },
+  {
+    recipient: 'zs13h7mt2xyk5t2xy38...2xvxser2',
+    amount: '-0.001',
+    status: 'Sent',
+    date: 'Jun 12, 2024',
+  },
+  {
+    recipient: 'zs13h7mt2xyk5t2xy38...2xvxser2',
+    amount: '-0.001',
+    status: 'Sent',
+    date: 'Jun 12, 2024',
+  },
+
+  {
+    recipient: 'u1jjfvrz0jt2xy38...a77mt2xy383x',
+    amount: '+0.000',
+    status: 'Sent',
+    date: 'May 11, 2024',
+  },
+  {
+    recipient: 'Shielded recipient',
+    amount: '+0.001',
+    status: 'Sent',
+    date: 'Mar 14, 2024',
+  },
+  {
+    recipient: 'Shielded recipient',
+    amount: '-0.001',
+    status: 'Received',
+    date: 'Dec 23, 2023',
+  },
+  {
+    recipient: 'Shielded Recipient',
+    amount: '+0.008',
+    status: 'Received',
+    date: 'Oct 31, 2023',
+  },
+];
```

Comment:
Hardcoded history data?

Reply:
The idea is to have a method to retrieve history from webzjs library. It's not implemented yet (even from zcash community perspective), so I keep it hardcoded for now